conf/denas/asr/supernet_large.conf

```diff
@@ -1,8 +1,22 @@
 SUPERNET:
```

Comment:
For the super_net large config file here, are you intend to do prune on large built-in supernet? 

please add explaination for key parameters in readme.

Reply:
The super_net large config file is to define the structure of super_net, and pruner will prune supernet to construct a smaller model.